### ISSUE TITLE
role="status" on alert-group component causing a11y violations

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-next#1562]
Should we just increase the a11y threshold to account for these, or is there another preferred approach?

<img width="802" alt="Screen Shot 2019-03-12 at 4 28 14 PM" src="https://user-images.githubusercontent.com/35148959/54237589-4d340d80-44e4-11e9-85a8-5368380f48b6.png">
<img width="824" alt="Screen Shot 2019-03-12 at 4 27 57 PM" src="https://user-images.githubusercontent.com/35148959/54237590-4dcca400-44e4-11e9-87c6-48fdf93657b7.png">

cc @jgiardino @seanforyou23

**Comment by mcoker** [comment]
from http://adrianroselli.com/2015/08/where-to-put-your-search-role.html

> Adding an ARIA role overrides the native role semantics in the accessibility tree which is reported via the accessibility API, and therefore ARIA indirectly affects what is reported to a screen reader or other assistive technology.

Should we wrap the alert group `ul` in `div role="status"`?

**Comment by jgiardino** [comment]
This is kind of a duplicate of https://github.com/patternfly/patternfly-next/issues/1375. But this issue is on hold until we implement the solution in react (i.e. I want to let the JS dev figure out the best way to load the live region on the page, then update core based on that).

For this issue, we could just remove `role="status"` since we plan to remove that anyway.
